Mobile/iOS/Templates/Release_3.4.9/xxx-thumbnails_not_downloaded.md

```diff
@@ -0,0 +1,48 @@
+|4| TIFF| 1. From web, upload a TIFF file<br>2. Open the app<br>3. Tap to download the file | 2. Thumbnail is shown<br>3. Thumbnail is shown|||
+|**Video Formats**|||||
+|5| AVI| 1. From web, upload a AVI file<br>2. Open the app<br>3. Tap to download the file | 2. Thumbnail is shown<br>3. Thumbnail is shown|||
+|6| MPG/MPEG/MP4 | 1. From web, upload a MPG/MPEG/MP4 file<br>2. Open the app<br>3. Tap to download the file | 2. Thumbnail is shown<br>3. Thumbnail is shown|||
+|7| MOV | 1. From web, upload a MOV file<br>2. Open the app<br>3. Tap to download the file | 2. Thumbnail is shown<br>3. Thumbnail is shown|||
+|**Actions**|||||
+|8|Load files| Upload an large amount of image files in a folder|All thumbnails are correcly shown|||
+|9|Move file| 1. Upload a image<br>2. Move the image to another folder|1. Thumbnail is shown <br>2. Thumbnail is shown in the new location|||
+|10|Rename file|  1. Upload a image<br>2. Rename the image using special characters|1. Thumbnail is shown <br>2. Thumbnail is shown with the new name|||
+|11|Duplicated name| 1. Upload an image<br>2. Rename another image with the same name in other folder <br>|1. Thumbnail is shown <br>2. Both thumbnails are correct|||
+|12|Overwrite file in mobile device| 1. Upload an image<br>2. Upload another image with the same name in the folder<br>3. Solve the conflict with "overwrite"|1. Thumbnail is shown <br>2. A conflict is detected<br>3. The thumbnail of the new image is shown |||
+|13|Overwrite file in server| 1. Switch the device connection off<br> 2. Upload an image to app<br>3. In server, upload another image with the same name<br>4. Switch the device connection on<br>5. Solve the conflict with "overwrite"|4. A conflict is detected<br>5. The thumbnail of device's version is shown |||
+|14|Move and overwrite file| 1. Upload an image<br>2. Upload another image with the same name in another folder<br>3. Move one image to the folder in which is the another one<br>4. Solve the conflict with "overwrite"|2. Both thumbnails are shown <br>3. A conflict is detected<br>4. The thumbnail of the second image is shown |||
+|15|Overwrite from a external app| 1. Upload an image<br>2. Upload another image from an external app with the same name<br>3. Solve the conflict with "overwrite"|1. Thumbnail is visible<br>2. A conflict is detected<br>3. The thumbnail of the second image is shown |||
+|16|Rename file (conflict)| 1. Upload an image<br>2. Upload another image with the same name in the folder<br>3. Solve the conflict with "rename"|1. Thumbnail is shown <br>2. A conflict is detected<br>3. Both thumbnails are shown |||
+|17|Delete file locally| 1. Upload an image<br>2. Download the image<br>3. Delete locally the image|1. Thumbnail is shown <br>2. Thumbnail is shown <br>3. Thumbnail is shown |||
+|18|Share file|1. Upload an image<br>2. Share the image to another user<br>3. Login with this user|1. Thumbnail is shown <br>3. Thumbnail is shown for the sharee|||
```

Comment:
Share test case is not very "important", so that the image is in the account whatever the owner is. i keep it anyway.